test/e2e/pkg/manifest.go

```diff
@@ -82,6 +82,11 @@ type Manifest struct {
 	LoadTxSizeBytes   int `toml:"load_tx_size_bytes"`
 	LoadTxBatchSize   int `toml:"load_tx_batch_size"`
 	LoadTxConnections int `toml:"load_tx_connections"`
+
+	// When true, enables Prometheus metrics for all nodes.
+	// When false, just apply the Prometheus flag of each individual node, if present.
+	// Defaults to false.
+	Prometheus bool `toml:"prometheus"`
```

Comment:
Have you tested the case when this is false, but the one added below is true for several nodes?
Won't there be a socket binding error?

Reply:
As discussed offline, I removed the config field for a node and kept just one top-level flag that applies to all the nodes. This will make things simpler for the user, as this seems to be the most typical scenario where the user just wants to enable the metrics for all nodes or none.